cmd/deploy.go

```diff
@@ -170,10 +171,31 @@ func setFlags(conf *clab.Conf) {
 	}
 }
 
-func createHostsFile(containers []types.Container, dirPath string, bridgeName string) error {
+func createHostsFile(containers []types.Container, bridgeName string) error {
 	if bridgeName == "" {
 		return fmt.Errorf("missing bridge name")
 	}
+	data := hostsEntries(containers, bridgeName)
+	if len(data) == 0 {
+		return nil
+	}
+	f, err := os.OpenFile("/etc/hosts", os.O_APPEND|os.O_WRONLY, os.ModeAppend)
+	if err != nil {
+		return err
+	}
+	defer f.Close()
+	_, err = f.WriteString("\n")
+	if err != nil {
+		return err
+	}
+	_, err = f.Write(data)
```

Comment:
do we want to check if the entry we about to add was already in the /etc/hosts?
can be a leftover from a failed destroy or a misconfig/clash of names.

Reply:
`data` here is all the entries as a single []byte.
adding entries one by one might lead to discrepancies. It's more simple to add all entries at once.
We might need a cleanup function at deploy. a function that deletes entries with the same prefix as the fabric to be deployed